api/src/main/java/ca/bc/gov/educ/studentdatacollection/api/rules/validationrules/impl/DuplicateEnrolledProgramRule.java

```diff
@@ -0,0 +1,57 @@
+package ca.bc.gov.educ.studentdatacollection.api.rules.validationrules.impl;
+
+import ca.bc.gov.educ.studentdatacollection.api.constants.StudentValidationFieldCode;
+import ca.bc.gov.educ.studentdatacollection.api.constants.StudentValidationIssueSeverityCode;
+import ca.bc.gov.educ.studentdatacollection.api.constants.StudentValidationIssueTypeCode;
+import ca.bc.gov.educ.studentdatacollection.api.constants.v1.CollectionTypeCodes;
+import ca.bc.gov.educ.studentdatacollection.api.rules.ValidationBaseRule;
+import ca.bc.gov.educ.studentdatacollection.api.service.v1.ValidationRulesService;
+import ca.bc.gov.educ.studentdatacollection.api.struct.StudentRuleData;
+import ca.bc.gov.educ.studentdatacollection.api.struct.v1.SdcSchoolCollectionStudentValidationIssue;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.core.annotation.Order;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+/**
+ *  | ID  | Severity | Rule                                                                  | Dependent On |
+ *  |-----|----------|-----------------------------------------------------------------------|--------------|
+ *  | V75 | ERROR    | List of enrolled programs codes must                                  | NONE         |
+ *                     not include duplicate program (e.g. 404005)
+ */
+@Component
+@Slf4j
+@Order(190)
+public class DuplicateEnrolledProgramRule implements ValidationBaseRule {
+    private final ValidationRulesService validationRulesService;
+    public DuplicateEnrolledProgramRule(ValidationRulesService validationRulesService) {
+        this.validationRulesService = validationRulesService;
+    }
+    @Override
+    public boolean shouldExecute(StudentRuleData studentRuleData, List<SdcSchoolCollectionStudentValidationIssue> validationErrorsMap) {
+        log.debug("In shouldExecute of DuplicateEnrolledProgramRule-V75: for collectionType {} and sdcSchoolCollectionStudentID :: {}", studentRuleData.getCollectionTypeCode(),
+                studentRuleData.getSdcSchoolCollectionStudentEntity().getSdcSchoolCollectionStudentID());
+        return CollectionTypeCodes.findByValue(studentRuleData.getCollectionTypeCode(), studentRuleData.getSchool().getSchoolCategoryCode()).isPresent()
+                && StringUtils.isNotEmpty(studentRuleData.getSdcSchoolCollectionStudentEntity().getEnrolledProgramCodes())
+                && isValidationDependencyResolved("V75", validationErrorsMap);
+    }
+
+    @Override
+    public List<SdcSchoolCollectionStudentValidationIssue> executeValidation(StudentRuleData studentRuleData) {
+        log.debug("In executeValidation of DuplicateEnrolledProgramRule-V75 for sdcSchoolCollectionStudentID ::" + studentRuleData.getSdcSchoolCollectionStudentEntity().getSdcSchoolCollectionStudentID());
```

Comment:
Do we mean DuplicateEnrolledProgramRule-V30?  I see it mapped here: https://github.com/bcgov/EDUC-STUDENT-DATA-COLLECTION-API/pull/343/commits/0ae101b97baae28a3dd1b32a0d8aeb5164e04814#diff-3666231be08e8d80d6dbf547ec77c047a7ef3368a9e2bdf0a481fad9b01b3f1bR22